src/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -58,11 +58,22 @@ const render = function render(interaction) {
 
         const placeholderText = interaction.attr('placeholderText');
 
+        const getItemLanguage = function getItemLanguage() {
+            let itemLang = window.document.documentElement.querySelector('.qti-item').getAttribute('lang');
```

Comment:
**question:** Why is it needed to use such a long chain? What will happen if one of the elements is not found?

**though:** I wonder how this would behave in the context of several items present in the same page.

**nitpick:** I think it would be better to use the parent lookup, which will use an ascendant search, instead of a descending approach that could introduce false-positives.

**suggestion:** I know that we should try to get rid of jQuery, but this is a case where it may be very convenient
```suggestion
            let itemLang = $container.closest('.qti-item').attr('lang');
```

Reply:
ok!